splash/browser_tab.py

```diff
@@ -946,9 +964,8 @@ def _set_request_headers(self, request, headers):
             headers = headers.items()
 
         for name, value in headers or []:
-            request.setRawHeader(to_bytes(name), to_bytes(value))
-            if name.lower() == 'user-agent':
-                self.set_user_agent(value)
+            name, value = to_bytes(name), to_bytes(value)
+            request.setRawHeader(name, value)
```

Comment:
`request.setRawHeader(to_bytes(name), to_bytes(value))` was not worse :) It had a tiny advantage of not mutating name and value.